provision/core/wp-cli/provision.sh

```diff
@@ -0,0 +1,30 @@
+wp_cli_setup() {
+  echo " "
+  echo " * Installing/updating WP-CLI"
+  # WP-CLI Install
+  local exists_wpcli
+
+  # Remove old wp-cli symlink, if it exists.
+  if [[ -L "/usr/local/bin/wp" ]]; then
```

Comment:
maybe move this to the deprecated file?

Reply:
I think it makes sense as WP CLI wasn't deprecated, this is just migration code